src/main/java/timl/Exeptions/EmptyException.java

```diff
@@ -0,0 +1,4 @@
+package timl.Exeptions;
```

Comment:
Names representing packages should be in all lower case.

https://se-education.org/guides/conventions/java/basic.html#naming